.travis.yml

```
@@ -32,11 +32,17 @@ jobs:
         - docker tag bitbots/bitbots-base:kinetic bitbots/bitbots-base:latest
         - docker push bitbots/bitbots-base:latest
     - stage: deploy
-      name: "Domestic: base image"
+      name: "Domestic base image"
       script:
         - docker build -f domestic/melodic/Base.docker -t bitbots/bitbots-domestic:melodic-base domestic/melodic
+        - docker images
         - docker push bitbots/bitbots-domestic:melodic-base
+      name: "Industrial base image"
```

There is a line missing: 

```suggestion
     - stage: deploy
      name: "Industrial base image"
```

---

I hope I did it right this time.

---

I think the `script` tag might be missing as well.